cabra/src/pages/components/RelatedTaskList.tsx

```diff
@@ -1,46 +1,51 @@
-import tw, { styled } from "twin.macro";
+import "twin.macro";
 
 import { DirectedRelationType } from "../../types/taskRelation";
 import RelatedTask from "./RelatedTask";
 import { getRelatedTasks } from "../../api/taskRelations";
+import { useNavigate } from "react-router-dom";
 import { useQuery } from "react-query";
 
-interface RelatedTasksListProps {
+interface Props {
   relationType: DirectedRelationType;
   parentTaskId: string;
+  className?: string;
 }
 
-const Container = styled.div(tw``);
-
 export default function RelatedTasksList({
   relationType,
   parentTaskId,
-}: RelatedTasksListProps) {
+  className,
+}: Props) {
   const { data, isLoading, error } = useQuery(
     getRelatedTasks.cacheKey(parentTaskId, relationType),
     () => getRelatedTasks.run(parentTaskId, relationType)
   );
+  const navigate = useNavigate();
 
-  if (isLoading) return <Container>Loading</Container>;
-  if (error) return <Container>Oops</Container>;
-  if (!data?.data) return <Container />;
+  if (isLoading) return <div>Loading</div>;
+  if (error) return <div>Oops</div>;
+  if (!data?.data) return <div />;
 
   const subtasks = data.data.slice().reverse();
 
   if (subtasks.length == 0) {
-    return <Container>{"<No tasks>"}</Container>;
+    return <div>{"<No tasks>"}</div>;
   }
 
   return (
-    <Container>
+    <div className={className}>
       {subtasks.map((relatedTask) => (
         <RelatedTask
           key={relatedTask.relationId}
           task={relatedTask.task}
```

Comment:
According to the most popular Airbnb style guide, we should use destructing here
```suggestion
      {subtasks.map(({ relationId, task }) => (
        <RelatedTask
          key={relatedTask.relationId}
          task={relatedTask.task}
```
but the change is optional